private-eye.js

```diff
@@ -6,11 +6,36 @@
     styles.innerHTML = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
     document.body.appendChild(styles);
 
+    var DEFAULT_MESSAGE = "This is a link to a private site, which may or may not be accessible to you.";
+    var hrefValue;
+    var linkMessage;
```

Comment:
Can we move these variable declarations into the callback below? No need to have them accessible outside of it or between iterations.